addons/workspace-dots/addon.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Editor workspace dots theme",
+  "description": "Allows the customisation of the style of the editor workspace dots.",
+  "tags": ["editor", "codeEditor", "theme"],
+  "versionAdded": "1.39.0",
+  "userscripts": [{ "url": "userscript.js", "matches": ["projects"] }],
+  "settings": [
+    {
+      "name": "Dots theme",
+      "id": "theme",
+      "type": "select",
+      "default": "dots",
+      "potentialValues": [
+        { "id": "dots", "name": "Default dots" },
+        { "id": "crosshairs", "name": "Crosshairs" },
+        { "id": "lines", "name": "Solid lines" },
+        { "id": "vertical", "name": "Vertical Lines" },
+        { "id": "horizontal", "name": "Horizontal Lines" },
+        { "id": "none", "name": "None" }
+      ]
+    },
+    {
+      "name": "Use a custom spacing value",
+      "id": "useSpacing",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Spacing value",
```

Comment:
This is confusing when combined with `script-snap`'s setting.

Reply:
Yes, yes it is, but it is the exact feature the original suggestion wanted, and it does have its place. I believe they wanted to mimic the style of scratch 1.4? But I do agree it is confusing, perhaps a notice or better wording?

Reply:
should script snapping get rid of the setting? however technically (other than design/looks) this addon is only useful when paired with script-snapping, but i might be wrong. we should discuss what should modify this scratch editor setting because it probably shouldnt be in 2 places, it should be consistent.

Reply:
wait... whats the difference between `spacing_` and `scale_`?

Reply:
The two settings technically do different things since this one is only for the appearance.

Reply:
The point of this setting is rather confusing but it's main use is to provide a grid that does NOT match up with the snapping addon, for visual purposes only, and does not affect snapping. The grid would become as useless as if you turned the grid off, which is also a visual choice

Reply:
this is weird and my brain hurts :P why would we not want to be consistent with the snapping?

Reply:
So it's possible to have a Scratch 1.x theme and still be able to snap normally. I'm not sure it's worth it though.

Reply:
@Samq64 can you expand bc im unfamilar with how scratch 1.x behavior is.

Reply:
https://en.scratch-wiki.info/w/images/Scratch_Interface.png

The workspace has a vertical line pattern.

Reply:
but you'd probably want the blocks to still align with a line, but maybe not just every line right? so maybe the setting could be something where it divides the snap spacing by a value?